test/data/usage_scenarios/schema_checker_valid.yml

```diff
@@ -16,6 +21,9 @@ flow:
   - name: Stress
     container: test-container
     commands:
-      - type: console
-        command: stress-ng -c 1 -t 1 -q
-        note: Starting Stress
\ No newline at end of file
+      - type: console 
+        command: stress-ng -c 1 -t 1 -q 
+        note: Starting Stress
+        shell: bash 
+        log-stdout: yes 
```

Comment:
Why is this valid? log-stdout is a boolean ...?

Reply:
I've changed this to a more standard 'true'